setup.py

```diff
@@ -51,7 +51,9 @@
         'Products.CMFCore',
         'setuptools',
         'ftw.upgrade',
-        'path.py<9.0'
+        'path.py<9.0',
+        'plone4.csrffixes',
```

Comment:
We have a lot of deployments which do not have `plone4.scrffixes` installed, on purpose. See:
- https://github.com/4teamwork/ftw-buildouts/pull/31
- https://github.com/4teamwork/matrix/pull/139

Therefore I would prefer not to have a dependency to `plone4.csrffixes` here.